examples/vue-webpack/src/App.vue

```diff
@@ -15,3 +20,30 @@ export default {
   },
 }
 </script>
+
+<style>
+.Example-List {
+  width: 235px;
+  outline: 1px #1162fe solid;
+  padding: 20px;
+  height: 400px;
+  display: -ms-flexbox;
+  display: -webkit-flex;
+  display: flex;
+  -webkit-flex-direction: column;
```

Comment:
Don't vue cli has autoprefixer enabled by default?

Reply:
I will remove prefixes

Reply:
I don't think that we need prefixes in sites for developers at all.